locales/client/en.json

```diff
@@ -775,7 +775,7 @@
     "title": "Support the movement",
     "label": {
       "tagline": "Buy our books.",
-      "description": "LikeCoin protocol is a movement to redistribute the income generated by contents licensed under Creative Commons. Our contents are free to use and our source codes are open source on <a href=\"https://github.com/likecoin\">Github</a>.<br/><br/><a href=\"http://sched.co/E6zv\">Our presentation in Creative Commons Summit 2018</a>",
+      "description": "LikeCoin protocol is a movement to redistribute the income generated by contents licensed under Creative Commons. Our contents are free to use and our source codes are open source on <a href=\"https://github.com/likecoin\">Github</a>. <br/><br/><a href=\"http://sched.co/E6zv\">Our presentation in Creative Commons Summit 2018</a>",
```

Comment:
What is the space for?

Reply:
Backer page use it as og:description. For better visualization when html tags are stripped out